test/functionalTest/cases/4495_servicepath_levels_with_zero_length/4495_servicepath_levels_with_zero_length.test

```diff
@@ -0,0 +1,266 @@
+# Copyright 2024 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Servicapath with zero length levels are not allowed
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create entity E1/T1/A1=1, with servicepath '/Level_1' (success case)
+# 02. GET /v2/entities/E1, with servicepath '/Level_1' to see E1/T1/A1=1
+# 03. Create entity E2/T2/A2=2, with servicepath '/Level_1/Level_2' (success case)
+# 04. GET /v2/entities/E2, with servicepath '/Level_1/Level_2' to see E2/T2/A2=2
+# 05. Create entity E3/T3/A3=3, with servicepath '//Level_1' (SP level with 0 length), see error
+# 06. Create entity E3/T3/A3=3, with servicepath '///Level_1///Level_2//' (SP level with 0 length), see error
+# 07. Create entity E3/T3/A3=3, with servicepath '/Level_1///Level_2' (SP level with 0 length), see error
+# 08. Create entity E3/T3/A3=3, with servicepath '/Level_1/Level_2//' (SP level with 0 length), see error
+#
```

Comment:
To complete the different case, I'd suggest to finally add these ones:

```
# 0x. Create entity E4/T4/A4=4, with servicepath '/Level_1/Level_2/' (success case)
# 0x. GET /v2/entities/E4, with servicepath '/Level_1/Level_2' (without slash) to see E4/T4/A4=4
```

Reply:
Fixed in 528f688